opg-base-1604/Makefile

```diff
@@ -1,9 +1,20 @@
 .PHONY: build
+SHELL = '/bin/bash'
 
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
+image := $(current_dir)
 registryUrl ?= registry.service.opg.digital
-
-image = opg-base-1604
+SEED := $(shell /bin/bash -c "echo $$RANDOM")
 
 build:
-	docker build $(no-cache) -t "$(registryUrl)/$(image)" .
-	# docker tag "$(registryUrl)/$(image)" "$(registryUrl)/$(image):$(newtag)"
+	docker-compose -f tests/docker-compose.yml build
+
+test:
+	echo "Seed: ${SEED}"
+	docker-compose -p "${SEED}" -f tests/docker-compose.yml up -d
+	inspec exec tests -t docker://${SEED}_basegen_1
```

Comment:
some slight fragility on relying on the container name being <seed>_basegen_1 

I recommend you extract the container name from docker-compose
```
docker-compose ps | grep basegen | awk '{ print $1 }'
```

Reply:
Agreed. If you've a good way of doing that. I'm all ears. 

I'm not sure it is super fragile though, as I always set that container name in the docker-compose files in the test directory.